browser/src/control/Control.LokDialog.js

```diff
@@ -660,6 +660,12 @@ L.Control.LokDialog = L.Control.extend({
 		// Override default minHeight, which can be too large for thin dialogs.
 		L.DomUtil.setStyle(dialogContainer, 'minHeight', height + 'px');
 
+		// Title bar may overflow due to range name. So we should have max width
+		var titleBar = dialogContainer.previousSibling;
+		var leftPadding = window.getComputedStyle(titleBar).getPropertyValue('padding-left').slice(0, -2);
```

Comment:
In CSS there are units with more than 2 (rem) and less than 2 characters (%).
https://www.w3schools.com/cssref/css_units.php

To make code bulletproof we should take it into account.
BTW I see that this titlebar is using em as units. Here we add this to px.
```
.ui-dialog .ui-dialog-titlebar {
    padding: 0.4em 1em;
```

Maybe as we do it programatically anyway, can we declare here some constant variable: `titlePadding`, which we will set here and on creation of the dialog? So we will be sure it is equal.

Reply:
Couldn't understand clearly but. Although units defined em here, "window.getComputedStyle(titleBar).getPropertyValue returns "px". One day it slight probability can return something else. But can't we trust the returned value now?